src/App.svelte

```diff
@@ -11,6 +13,36 @@
 
   let component;
   let params;
+  let links = [];
+
+  afterUpdate(() => {
+    const { app, ping, metric, table } = params;
+
+    links = [
+      ...(app
+        ? [
+            { url: "/", name: "apps" },
+            { url: `/apps/${app}/`, name: app },
+          ]
+        : []),
+      ...(ping ? [{ url: `/apps/${app}/pings/${ping}/`, name: ping }] : []),
+      ...(metric
+        ? [{ url: `/apps/${app}/metrics/${metric}/`, name: metric }]
+        : []),
+      ...(table
+        ? [
+            {
+              url: `/apps/${app}/pings/${table}/`,
+              name: table,
+            },
+            {
+              url: `/apps/${app}/tables/${table}/`,
+              name: `${table} table`,
+            },
+          ]
+        : []),
+    ].filter(Boolean);
```

Comment:
I am pretty sure this filter should no longer be necessary?

```suggestion
    ]
```